plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/DecoratedHiveMetastoreModule.java

```diff
@@ -56,7 +68,9 @@ protected void setup(Binder binder)
         newExporter(binder).export(HiveMetastoreFactory.class)
                 .as(generator -> generator.generatedNameOf(CachingHiveMetastore.class));
 
-        newSetBinder(binder, Procedure.class).addBinding().toProvider(FlushHiveMetastoreCacheProcedure.class).in(Scopes.SINGLETON);
+        if (installFlushMetadataCacheProcedure) {
+            newSetBinder(binder, Procedure.class).addBinding().toProvider(FlushHiveMetastoreCacheProcedure.class).in(Scopes.SINGLETON);
```

Comment:
pre-existing : Is `io.trino.plugin.hive.metastore.procedure` the right package for `FlushHiveMetastoreCacheProcedure` or rather `io.trino.plugin.hive.procedure`

cc @aczajkowski @findepi

Reply:
`DecoratedHiveMetastoreModule` exists in `io.trino.plugin.hive.metastore.procedure` package. The current package is the right place in my opinion.